RELEASE_NOTES.md

```diff
@@ -81,19 +81,23 @@ you are strongly advised to take a backup first.
   along with the traditional color-picker for manual sampling of neutral colors.
 
   Color calibration internally implements gamut-mapping and gamut-clipping,
-  which attempts to preserve the hue and luminance of pixels while avoiding imaginary and
-  out-of-gamut colors at the beginning of the pipeline, to improve the robustness of
+  which attempt to preserve the hue and luminance of pixels while avoiding imaginary and
+  out-of-gamut colors at the beginning of the pipeline, in order to improve the robustness of
   color-grading modules inside the pipeline. This is made necessary because
   white balancing will only push input out-of-gamut colors farther away,
-  and will noticeably help dealing with artifical colored lights (LED, stage lighting, etc.).
+  and gamut-mapping and -clipping will noticeably help dealing with artifical colored
+  lights (LED, stage lighting, etc.).
```

Comment:
I think without repeating the subjects, the 'will noticeably help' part could be interpreted as having 'white balance' as the subject.

Reply:
you are repeating **and** here, not really good to me but I'm not native English.

Reply:
We don't really do the `-clipping` style of things in English. 

You should say `gamut-mapping and gamut-clipping` or `gamut-mapping/clipping`.

Reply:
could even live without the hyphen